frontend/src/pages/Home.jsx

```diff
@@ -262,6 +272,7 @@ export default function Home() {
               <button
                 className="select-button"
                 type="button"
+                // onClick={() => counterDivToggle()}
```

Comment:
Commented code here

Reply:
Yes, I missed this one, it has been deleted.

Reply:
Ok, great, keep me posted !